src/jolt/vm/mod.rs

```diff
@@ -224,50 +226,23 @@ pub trait Jolt<F: PrimeField, G: CurveGroup<ScalarField = F>> {
       }
     }
 
-    let E_poly: DensePolynomial<F> = DensePolynomial::from_u64(&read_values);
-    let dim: DensePolynomial<F> = DensePolynomial::from_usize(access_sequence);
-    let read_cts: DensePolynomial<F> = DensePolynomial::from_usize(&read_cts);
-    let final_cts: DensePolynomial<F> = DensePolynomial::from_usize(&final_cts);
-    let init_values: DensePolynomial<F> = DensePolynomial::from_u64(program_code);
-    unimplemented!("commit to these polynomials");
-
-    let init_poly = DensePolynomial::new(
-      (0..code_size)
-        .map(|i| {
-          // addr is given by i, init value is given by program_code, and ts = 0
-          hash_func(&F::from(i as u64), &F::from(program_code[i]), &F::zero())
-        })
-        .collect::<Vec<F>>(),
-    );
-
-    let read_poly = DensePolynomial::new(
-      (0..m)
-        .map(|i| hash_func(&F::from(read_addrs[i] as u64), &E_poly[i], &read_cts[i]))
-        .collect::<Vec<F>>(),
-    );
-
-    let write_poly = DensePolynomial::new(
-      (0..m)
-        .map(|i| {
-          hash_func(
-            &F::from(read_addrs[i] as u64),
-            &F::from(read_values[i]),
-            &(read_cts[i] + F::one()),
-          )
-        })
-        .collect::<Vec<F>>(),
-    );
-
-    let final_poly = DensePolynomial::new(
-      (0..code_size)
-        .map(|i| {
-          // addr is given by i, init value is given by program_code, and ts = 0
-          hash_func(&F::from(i as u64), &F::from(program_code[i]), &final_cts[i])
-        })
-        .collect::<Vec<F>>(),
-    );
-
-    unimplemented!("memory checking");
+    let E_poly: DensePolynomial<F> = DensePolynomial::from_u64(&read_values); // v_ops
+    let dim: DensePolynomial<F> = DensePolynomial::from_usize(access_sequence); // a_ops
+    let read_cts: DensePolynomial<F> = DensePolynomial::from_usize(&read_cts); // t_read
+    let final_cts: DensePolynomial<F> = DensePolynomial::from_usize(&final_cts); // t_final
+    let init_values: DensePolynomial<F> = DensePolynomial::from_u64(program_code); // v_mem
```

Comment:
we might as well move everything in this function up to here into `PCPolys::new`

Reply:
Done.